sources/platform/integrations/programming/api.md

```diff
@@ -40,8 +46,73 @@ do not require an authentication token because they contain a hard-to-guess iden
 
 When working under an organization account, you will see two types of API tokens on the Integrations page.
 
-![Integrations page in the Apify Console in organizatoin mode](../images/api-token-organization.png)
+![Integrations page in the Apify Console in organization mode](../images/api-token-organization.png)
 
 The Personal API tokens are different from your own Personal API tokens mentioned above. If you use this token in an integration, it will have the same permissions that you have within the organization, and all the operations you use it for will be ascribed to you.
 
 On the other hand the Organization API tokens (only visible if you are the owner or have Manage access tokens permission) have full permissions and are not tied to a specific member of the organization.
+
+## API tokens with limited permissions {#limited-permissions}
+
+By default, tokens can access all data in your account. If that is not desirable, you can choose to limit the permissions of your token, so that it can only access data needed for the particular use case. We call these tokens **scoped**.
+
+**A scoped token can access only those resources that you'll explicitly allow it to.**
+
+### How to create a scoped token
+
+Scoped tokens behave like standard API tokens and are managed through the [Integrations](https://console.apify.com/settings/integrations) page in Apify Console. When creating a token (or updating an existing one), simply toggle "Limit token permissions" to make the token scoped.
+
+Once the token is scoped, you can specify the token's permissions.
+
+![Toggle "Limit token permissions" to make a token scoped](../images/tmp__create-scoped-api-token.png)
+
+### Account level vs resource-specific permissions
+
+We support two different types of permissions for tokens:
+
+- **Account level permissions**: These will apply to all resources in the entire account. For example, you can use these to allow the token to run _all_ your Actors.
+
+- **Resource-specific permissions**: These will apply only to specific, existing resources. For example, you can use these to allow the token to read from a particular dataset.
+
+:::tip
+A single token can combine both types. You can create a token that can _read_ any data storage, but _write_ only to one specific dataset.
+:::
```

Comment:
```suggestion
:::tip Combined token permissions

A single token can combine both types. You can create a token that can _read_ any data storage, but _write_ only to one specific dataset.

:::
```

Reply:
Are these callout titles a strong requirement?

TBH, I don't see that much value in them... ideally there'd be no title, just the text plus the icon, but since the title has to be there, for me "Tip" / "Note" / "Important" work better. The proposed titles seem very vague and non-informative to me.

Reply:
In my experience documentation is more scanned than read. These admonitions and titles within them, can help grab attention as well as at a glance help user decide whether he found what was looking for or not. Moreover that is also how we are doing them anywhere else in docs. I would be in favour of keeping the title, but can definitely come up with few alternatives if these seem to vague

Reply:
> In my experience documentation is more scanned than read.

Totally on board with this :+1: 

But IMHO the generic titles such as "important" or "tip" can actually greatly improve scannability: If something is a tip, I know I can skip it, if it's important, I should probably read it etc.

My argument is basically that these generic titles might bring more value than callout-specific, yet non-informative titles.

Btw Docusaurus itself [combines both approaches](https://docusaurus.io/docs/next/markdown-features/toc). Specific titles where it makes sense, generic where it doesn't.

I'll also try to think about alternatives for specific titles, but I'd actually prefer to keep the generic ones as fallback.